newm-server/src/main/kotlin/io/newm/server/features/minting/repo/MintingStatusHistoryRepository.kt

```diff
@@ -0,0 +1,12 @@
+package io.newm.server.features.minting.repo
+
+import io.newm.server.features.minting.database.MintingStatusTransactionEntity
+import io.newm.server.features.minting.model.MintingStatusTransactionModel
+import io.newm.server.typealiases.SongId
+import java.util.UUID
+
+interface MintingStatusHistoryRepository {
```

Comment:
This could be part of the existing `MintingRepository`, rather than creating a new one.

Reply:
Can you elaborate more on the benefit of having it in the same repository? I'm afraid that we are breaking Separate Of Concern design pattern by merging 2 functionalities in one class.